code/modules/reagents/chemistry/reagents/other.dm

```diff
@@ -27,7 +27,7 @@
 	color = "#0064C8" // rgb: 0, 100, 200
 	overdose_threshold = REAGENTS_OVERDOSE * 2
 	custom_metabolism = REAGENTS_METABOLISM * 5 //1.0/tick
-	purge_list = list(/datum/reagent/toxin, /datum/reagent/medicine, /datum/reagent/consumable)
+	purge_list = list(/datum/reagent/toxin, /datum/reagent/medicine, /datum/reagent/consumable, /datum/reagent/zombium)
```

Comment:
this really, really weakens zombium, since you now have an infinite, readily accessable reagent to purge it. Not quite sure why this would be required tbh.

Reply:
marines already have a cakewalk vs zombies, i'd rather shift the systems so that factions without infinite meds and bullets can have a chance at fighting zombies.
water has the downside of purging chems which can be a big deal if you're surviving off of one bicardine injector you found as a survivor

Reply:
> this really, really weakens zombium, since you now have an infinite, readily accessable reagent to purge it. Not quite sure why this would be required tbh.

Neurokiller Chemerettes are already free and unlimited, and on the few zombie rounds i've seen, the toxin first aid kits Rye packets were untouched. It's really just a marine skill issue if they turn.